### DISCUSSION TITLE
Minimum range of Y axis

### DISCUSSION THREAD

**Discussion by jrlacharnay** [RomRider/apexcharts-card#262] [Q&A]
Hi there,

In the [Mini Graph Card](https://github.com/kalkih/mini-graph-card), there is an option "min_bound_range" to avoid that the auto scaling of the Y axis zoom too much on the data.

Can I achieve the same with the Apex Charts card?

Thanks,
JR

**Comment by hiatsu0** [comment]
Was trying to find this option too, in order to make my auto scaling graphs look less dramatic. But apparently there is no way to set some minimum range for y axis?
